lib/classes/Utils.test.js

```diff
@@ -21,7 +21,7 @@ describe('Utils', () => {
   beforeEach(() => {
     serverless = new Serverless();
     utils = new Utils(serverless);
-    serverless.init();
+    return serverless.init();
```

Comment:
Shouldn't we put this right at the beginning of the `beforeEach` block like it's done below?

Reply:
You mean to move it before `utils = new Utils(serverless);` line? It can't be starting line here, as there's no `serverless` instance yet